src/requirements/PoapVoice/PoapVoiceForm.tsx

```diff
@@ -132,7 +132,7 @@ const PoapVoiceForm = ({ onClose, poapId }): JSX.Element => {
         </FormControl>
         <FormControl
           isRequired
-          isInvalid={!!errors?.voiceRequirement?.percentOrMinute}
+          isInvalid={!!(errors?.voiceRequirement as any)?.percentOrMinute}
```

Comment:
I think we're safe to remove the `any` cast inside this file too. I left some more detailed comments on this inside the `TwitterUrlInput.tsx` file